content/client-server-api/modules/spaces.md

```diff
@@ -87,10 +87,9 @@ the state of `#space:example.org` would consist of:
 }
 ```
 
-No state events in the child rooms themselves would be required (though they
-can also be present). This allows for users
-to define personal/private spaces to organise their own rooms without needing explicit
-permission from the room moderators/admins.
+No state events in the child rooms themselves would be required (though they can also
+be present). This allows for users to define spaces without needing explicit permission
+from the room moderators/admins.
```

Comment:
The spec defines neither "personal" nor "private" for rooms or spaces. The different dimensions of "privacy" seem irrelevant here though. The argument is merely that the state events in child rooms are optional so that you can build spaces without being an admin in all their rooms.